examples/chat/kube_deploy.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Path to the app config.
+appConfig: weaver.toml
+
+# Registry to upload the app docker image.
+repo: docker.io/my_docker_hub_username/
+
+listeners:
+- name: chat
+  public: true
+
+# Check `weaver kube deploy -h` to see which other options you can configure.
```

Comment:
Since we duplicate this here and in collatz/weaver.toml, maybe just say something like `Check weaver kube deploy -h to see which other options you can configure, and see collatz/weaver.toml for an example`?

Reply:
Done